app/controllers/laa_references_controller.rb

```diff
@@ -26,16 +48,26 @@ def laa_reference_params
     params.permit(
       :id,
       :defendant_id,
-      :maat_reference
+      link_attempt: %i[id defendant_id maat_reference defendant_identifier]
```

Comment:
still a bit confusing why we need `:defendant_id` twice and then alsow have a `:defendant_identifier` - could rename defendant_identifier to exactly what it is - `defendant_asn_or_nino` - we are going to need to remove this part at some point innay events as CP does not guaruantee either ASN or NINO will exist, will all have to be id

Reply:
👍 I have remove defendant_id from laa_reference_params and renamed defendant_identifier to defendant_asn_or_nino. I have also removed `f.hidden_field(:id), params[:id] ` from the form and replaced with `f.hidden_field(:defendant_asn_or_nino, value: (params[:id] || @defendant_asn_or_nino))`